local/http/http.go

```diff
@@ -212,6 +213,10 @@ func (s *Server) ProxyHandler(w http.ResponseWriter, r *http.Request) {
 
 // Handler handles HTTP requests
 func (s *Server) Handler(w http.ResponseWriter, r *http.Request) {
+	if s.AllowCORS {
+		w.Header().Add("Access-Control-Allow-Origin", "*")
```

Comment:
I'm wondering if we should not add default CORS values (see what Gorillw used to have: https://github.com/gorilla/handlers/blob/546854cf1d61a016260b224e2526a29ff5b5c5ae/cors.go#L28-L32)

Reply:
We should also be careful not to override header sent by the upstream app

Reply:
no this would not work because at this stage the header will never be already defined because the upstream server has not been contacted yet (contacting the upstream server is done in `s.Callback`)